src/main/kotlin/lotto/domain/PublishLotto.kt

```diff
@@ -0,0 +1,7 @@
+package lotto.domain
+
+class PublishLotto(val manualLottos: List<Lotto>, val autoLottos: List<Lotto>) {
+    val allLotto: List<Lotto> = manualLottos + autoLottos
```

Comment:
```suggestion
    val allLotto: List<Lotto>
        get()= manualLottos + autoLottos
```
위 코드와의 차이점을 알고 계실까요 ?